packages/vite-plugin-nitro/src/lib/vite-plugin-nitro.ts

```diff
@@ -119,3 +125,14 @@ export function nitro(options?: Options, nitroOptions?: NitroConfig): Plugin {
     },
   };
 }
+
+function isEmptyPrerenderRoutes(options?: Options): boolean {
+  if (!options || isArrayWithElements(options?.prerender?.routes)) {
+    return false;
+  }
+  return !options.prerender?.routes;
+}
+
+function isArrayWithElements<T>(arr: unknown): arr is [T, ...T[]] {
```

Comment:
TBH I would put this function into some utils library, but I don't think that it exists yet.